mozci/push.py

```diff
@@ -1169,6 +1205,9 @@ def classify_regressions(
             for g in all_groups
             if g.name not in list(bugbug_selection["groups"].keys())
         }
+
+        groups_still_running = {g.name for g in all_groups if self.is_group_running(g)}
```

Comment:
We should actually check if the group is still running in any of the parents of the push.
We iterate the parents and we either stop if we find the group run, or add the group to the groups_still_running set.
Pseudocode:
```Python
for group in real_failures:
    for parent in push.parents:
        # If the group run in a parent, then we don't care if it's still running in a grandparent.
        if group in parent.group_summaries:
            break

        if parent.is_group_running(group):
            groups_still_running.add(group)
            break

real_failures -= groups_still_running
unknown_failures |= groups_still_running
```